usr/share/rear/output/default/94_grub2_rescue.sh

```diff
@@ -110,6 +110,9 @@ else
     grub-mkconfig -o $grub_conf
 fi
 
+#Finding UUID of filesystem containing /boot 
+grub_boot_uuid=$(df /boot | awk 'END {print $1}' | xargs blkid -s UUID -o value)
```

Comment:
@schabrolles You could make the test easier (and we use this a lot in rear):
````
ls -ld /boot/grub* > /dev/null 2>&1
StopIfError "/boot is empty, or may be not mounted..."
````
Thank you for reviewing the code once more.